<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetSourceTemplate</class>
 <widget class="QWidget" name="WidgetSourceTemplate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>130</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>130</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Fuente</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1" colspan="2">
    <widget class="QLabel" name="LabelSourceType">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Tipo Fuente</string>
     </property>
     <property name="margin">
      <number>2</number>
     </property>
     <property name="indent">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="LabelSource">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QCheckBox" name="CheckBoxIgnoreSource">
     <property name="text">
      <string>No considerar esta fuente</string>
     </property>
     <property name="icon">
      <iconset theme="edit-delete">
       <normaloff/>
      </iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_5">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Estado</string>
     </property>
     <property name="margin">
      <number>2</number>
     </property>
     <property name="indent">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="LabelState">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Dispositivo</string>
     </property>
     <property name="margin">
      <number>2</number>
     </property>
     <property name="indent">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QToolButton" name="ToolButtonSelectSource">
     <property name="text">
      <string>Seleccionar</string>
     </property>
     <property name="icon">
      <iconset theme="document-open">
       <normaloff/>
      </iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QToolButton" name="ToolButtonState">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset theme="dialog-warning"/>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
